app/src/main/java/com/asfoundation/wallet/ui/iab/LocalPaymentPresenter.kt

```diff
@@ -196,6 +197,15 @@ class LocalPaymentPresenter(private val view: LocalPaymentView,
     }
   }
 
+  private fun handleSupportClicks() {
+    disposables.add(Observable.merge(view.getSupportIconClicks(), view.getSupportLogoClicks())
+        .throttleFirst(50, TimeUnit.MILLISECONDS)
+        .flatMapCompletable { localPaymentInteractor.showSupport(gamificationLevel) }
+        .subscribeOn(viewScheduler)
```

Comment:
i think this observe on is unnecessary since the method is always called in the main thread